src/shared/dissect-image.c

```diff
@@ -191,14 +191,10 @@ static int make_partition_devname(
 }
 #endif
 
-int dissect_image(
-                int fd,
-                const char *image_path,
+int dissect_loop_device(
```

Comment:
as i mentoined in that other PR: I am not convinced this is desirable: with this change we can only dissect stuff with privileges. Before we could analyze regular files too, without loopback devices, without privileges. I'd really like to keep that, and make this something we make use of in tests...

Reply:
Oh, was it possible to call this without privileges? I did not know that.

Reply:
At least on the current git HEAD, `dissect_image()` does not work for file descriptor for a regular file. Also, not work for block devices without privilege.
I guess we can modify the logic to support regular file or without privilege, but l'd like to do that later.